Utils.py

```diff
@@ -677,7 +677,7 @@ def read_snes_rom(stream: BinaryIO, strip_header: bool = True) -> bytearray:
 _faf_tasks: "Set[asyncio.Task[None]]" = set()
 
 
-def async_start(co: Coroutine[None, None, None], name: Optional[str] = None) -> None:
+def async_start(co: Coroutine[typing.Any, typing.Any, bool], name: Optional[str] = None) -> None:
```

Comment:
I don't understand this typing change.
If this is "fire-and-forget", that means nothing is ever done with the return value. So why are we saying the return value has to be `bool`?
Also because this is "fire-and-forget" and there is no later access to the task, the send type and yield type would have to be `None`.

Reply:
Actually, `Coroutine[None, None, Any]` could be a problem, because if the coroutine is yielding `None`, that task would get stuck in `_faf_tasks` forever, and would be a memory leak.

Better would be:
`Coroutine[Never, Never, Any]`

( `from typing_extensions import Never` )